modulefiles/gsi_hera.intel.lua

```diff
@@ -1,24 +1,23 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/miniconda3/modulefiles")
```

Comment:
Does this `prepend_path` point at the official installation all UFS and EMC apps should use?

Reply:
Others should also confirm but the `MODULEPATH`s I see in this PR jive with the ones that ufs-weather-model moved to (https://github.com/ufs-community/ufs-weather-model/issues/1465) and which global-workflow and other components are also moving to. The ufs-weather-model issue I refer to has a nice listing at the top of the various machine installs.

Reply:
The updated prepend_path and module versions all come from ufs-weather-model develop for Hera, Jet, and Orion.
I did not change setup for other machines because I cannot access them. But the ufs-weather-model develop still use intel 19 for wcoss2, which should be updated for both model and GSI.

Reply:
Looks like all of the (fully) supported R&D platforms are covered in this PR (Hera, Orion, Jet). @DavidHuber-NOAA is providing updates for S4 (low support). WCOSS2 is not considered in this PR since it's only intel 2019 still and not EPIC-maintained hpc-stacks. Will update WCOSS2 to intel 2022 at a later date (likely when we move to spack-stack but not 100% on that). Updates in this PR should not break WCOSS2 functionality.

Reply:
Note...support for Hercules will be coming soon (port just starting). We (g-w) are waiting for EPIC hpc-stack installs there. This PR should make it easier to add Hercules support via EPIC hpc-stacks.

Reply:
What's the timeline for intel2022 on WCOSS2?   I see `intel-classic/2022.2.0.262` and `intel-oneapi/2022.2.0.262` as modules available on Cactus.   

It would be nice for GSI to check off intel2022 on all supported platforms.   Building and running `gsi.x` and `enkf.x`  with intel2022 on WCOSS2 might identify new problems which impact other platforms.

Reply:
I think EMC is in the process of getting spack-stack v1.3.0 installed and tested with the WM on WCOSS2 now. We are waiting for that to be completed before switching the WM modules over to spack-stack and intel 2022.

Reply:
@natalie-perlin confirmed these installations are all correct and provided an updated location for the EPIC-managed HPC-stack on Cheyenne.  Getting this to install on Gaea is still an issue, but she is also working on that and will update the module file if it works.

Reply:
Paths look correct. A couple versions are ahead of where workflow is (and not in UFS), but we can update to match.

Reply:
Is this resolved, then?

Reply:
If GSI paths are ahead of g-w, will `gsi.x` and `enkf.x` run correctly in g-w?